pkgs/build-support/docker/examples.nix

```
@@ -715,4 +715,118 @@ rec {
     config = {
     };
   };
+
+  nix-shell-basic = buildNixShellImage {
```

```suggestion
  nix-shell-basic = streamNixShellImage {
```

(here and all instances below)

This avoids putting redundant docker image data in the NixOS binary cache.